src/utils/Options.ts

```diff
@@ -110,13 +110,23 @@ module n3Charts.Utils {
       return !(value === !defaultValue);
     }
 
-    static getNumber(value: any, defaultValue: number = 0) {
+    static getNumber(value: any, defaultValue: number) {
       var n = parseFloat(value);
       return !isNaN(n) ? n : defaultValue;
     }
 
-    static getString(value: any) {
-      var s = String(value);
+    static getDate(value: any, defaultValue: Date) {
+      return value instanceof Date ? value : defaultValue;
+    }
+
+    static getString(value: any, defaultValue?: string) {
+      var s = undefined;
+      if (value === undefined) {
+        s = String(defaultValue);
+      } else {
+        s = String(value);
+      }
+
       return s;
     }
```

Comment:
Right, that's way better. Dunno why I didn't write this. I'll change it right away.